src/jit/codegenarm64.cpp

```diff
@@ -416,13 +416,17 @@ void CodeGen::genSaveCalleeSavedRegistersHelp(regMaskTP   regsToSaveMask,
                                               int         lowestCalleeSavedOffset,
                                               int         spDelta)
 {
+    assert(spDelta <= 0);
     unsigned regsToSaveCount = genCountBits(regsToSaveMask);
     if (regsToSaveCount == 0)
     {
+        if (spDelta != 0)
+        {
+            genStackPointerAdjustment(spDelta, REG_NA, nullptr);
+        }
```

Comment:
oh, it's for vararg registers, but that still has a relatively small, fixed range that could be asserted. Maybe a comment should indicate that, here or elsewhere?

---
In reply to: [56382253](https://github.com/dotnet/coreclr/pull/3761#discussion_r56382253) [](ancestors = 56382253)